examples/js/README.md

```diff
@@ -0,0 +1,5 @@
+# RainCatcher JavaScript usage example
```

Comment:
:+1: it was originally `js-usage`, but I thought maybe having just js would be better. Will change